docs/CHANGELOG.md

```diff
@@ -7,6 +7,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Full EIP1559 Support
+
+Chainlink now includes experimental support for submitting transactions using type 0x2 (EIP-1559) envelope.
+
+EIP-1559 mode is off by default but can be enabled on a per-chain basis or globally.
+
+To enable globally, set `EVM_EIP1559_DYNAMIC_FEES=true`. Set with caution, if you set this on a chain that does not actually support EIP-1559 your node will be broken.
+
+In EIP-1559 mode, the total price for the transaction is the minimum of base fee + tip cap and fee cap. More information can be found on the [official EIP](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md).
+
+Chainlink's implementation of this is to set a large fee cap and modify the tip cap to control confirmation speed of transactions. So, when in EIP1559 mode, the tip cap takes the place of gas price roughly speaking.
+
+This should help to save gas on spikes: Chainlink will react faster on the upleg and avoid overpaying on the downleg.
```

Comment:
I think this is a little confusing - we don't save gas on the upleg right?

Reply:
It's actually quite difficult to predict. We might save some gas on the upleg too because we respond to tip, not the total, so as the base fee increases we dont unnecessarily boost our payment.

In the end, the only way to know will be to test it out in prod.